runtime/src/lib.rs

```diff
@@ -552,6 +585,9 @@ construct_runtime!(
 		CumulusXcm: cumulus_pallet_xcm::{Pallet, Event<T>, Origin} = 32,
 		DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>} = 33,
 
+		// Inflation.
+		Inflation: pallet_inflation::{Pallet, Call, Storage}  = 50,
```

Comment:
We should assign proper number here, because we added it between 33 and 40, it should be 34, but I would suggest to move it above under monetary dependencies and add number 12